spec/models/billing_subscription_spec.rb

```diff
@@ -9,7 +9,7 @@
       it 'calls Nonprofit#clear_cache after create' do
         billing_subscription = build(:billing_subscription)
         np = billing_subscription.nonprofit
-        expect(np).to receive(:clear_cache).once
+        expect(np).to receive(:clear_cache).thrice # once for nonprofit, once for billing subscription, once for billing plan
```

Comment:
`after_save` seems to work a bit differently in Rails 5.2. Previously it seemed to only fire the `after_save` on saving the Nonprofit. Now it does it on each of the items that need to be cached. It shouldn't cause any significant problem though, just slightly less efficient.

Reply:
Also want to be sure we don’t have any extraneous jobs or anything fired after save (if it happens more than once…)

Reply:
Agreed